### PR TITLE
Np 161&162 &163

### PR DESCRIPTION
**Pull request by maxkanin**
## **Ticket**

**JIRA ID:** [NP-161](https://parking-system.atlassian.net/browse/NP-161)
[NP-162](https://parking-system.atlassian.net/browse/NP-162)
[NP-163](https://parking-system.atlassian.net/browse/NP-163)

## Changes proposed in this pull request
- [IMPORTANT] Reviewer **will not review** any PR without summary of changes
- Move pay the bill page into payment main page
- Add logic when credit is not enough in pay the bill page
- Re-design top up button UI

please fill all the required fields from below Checklist otherwise reviewer **will not review** this PR
## Checklist
- [x] Screenshot evidence for both IOS and Android
- [x] Total change is less than 500 lines of code

## **Screenshot Evidence**

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/12dba430-696f-4fa6-9aba-0f3dcb47efc7

https://github.com/ParkingSystemWithLPR/Napark-application/assets/88888222/705b3bf3-4683-458a-b2c5-bf2c759e2e96

[NP-161]: https://parking-system.atlassian.net/browse/NP-161?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NP-162]: https://parking-system.atlassian.net/browse/NP-162?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[NP-163]: https://parking-system.atlassian.net/browse/NP-163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: screens/payment/Payment.tsx [hunk 1]
```diff
@@ -17,46 +22,92 @@ export type PaymentProps = CompositeScreenProps<
...
+            <View style={styles.innerContainer}>
+              <View>
+                <HeaderText
+                  text={item.licensePlate}
+                  textStyle={styles.licenseText}
+                />
+                <SubHeaderText text={item.province} />
+              </View>
+              <HeaderText
+                text={`${item.total} ฿`}
```
Comment (Petoxic) [inline screens/payment/Payment.tsx]: Use ฿ item.total instead for consistency

File: screens/payment/Payment.tsx [hunk 2]
```diff
@@ -17,46 +22,92 @@ export type PaymentProps = CompositeScreenProps<
...
+          </View>
+          <View
+            style={{
+              flexDirection: "row",
+              justifyContent: "space-between",
+              alignItems: "center",
+            }}
+          >
+            <HeaderText
+              text={`${mockBalance} ฿`}
```
Comment (Petoxic) [inline screens/payment/Payment.tsx]: This one also

File: screens/payment/PaymentSummary.tsx [hunk 3]
```diff
@@ -129,15 +139,15 @@ const PaymentSummary: React.FC<PaymentSummaryProps> = ({ navigation }) => {
...
           })}
           {renderPrice({
             attribute: "Tax",
-            value: "$10",
+            value: `mock value`,
           })}
           {renderTotal({
             attribute: "TOTAL",
-            value: "$60",
+            value: `${paymentDetail.total} ฿`,
```
Comment (Petoxic) [inline screens/payment/PaymentSummary.tsx]: this one also

File: screens/payment/Payment.tsx (hunk 1 above)
Comment (maxkanin) [reply screens/payment/Payment.tsx]: done

File: screens/payment/Payment.tsx (hunk 2 above)
Comment (maxkanin) [reply screens/payment/Payment.tsx]: done

File: screens/payment/PaymentSummary.tsx (hunk 3 above)
Comment (maxkanin) [reply screens/payment/PaymentSummary.tsx]: done

**Pull request closed without merging**